psi4/src/psi4/libmints/mintshelper.cc

```diff
@@ -161,6 +165,12 @@ void MintsHelper::common_init()
     if (print_)
         basisset_->print_detail();
 
+    // How many threads?
+    nthread_ = 1;
+    #ifdef _OPENMP
+        nthread_ = omp_get_max_threads();
```

Comment:
Agreed, but make sure it still obeys the value from set_num_threads() in this class.